data/mods/Generic_Guns/firearms/pistol.json

```
@@ -0,0 +1,66 @@
+[
+  {
+    "id": "pistol_levergun",
+    "copy-from": "henry_big_boy",
+    "type": "GUN",
+    "name": "cowboy carbine",
+    "ammo": "ammo_pistol",
+    "description": "Since the early days of western expansion, when ammunition supply was infrequent, shooters sought to share ammo between their sidearms and long arms.  This tube magazine lever action carbine allows the shooter to reduce the number of calibers carried and squeeze more power from their pistol ammunition.",
+    "clip_size": 10
+  },
+  {
+    "id": "pistol_machine",
+    "copy-from": "glock_18c",
+    "type": "GUN",
+    "name": "machine pistol",
+    "ammo": "ammo_pistol",
+    "description": "This pistol is a tiny machinegun you can stuff into a holster, with which you could dump its magazine at a blistering rate into any close range foes.  Machine pistols mostly see use by vehicle crewmen or bodygaurds of VIPs. Due to its preposterous rate of fire it is difficult to fire controllably.",
+    "magazines": [ [ "ammo_pistol", [ "pistol_mag",  "pistol_smg_mag" ] ] ]
+  },
+  {
+    "id": "pistol_medium",
+    "copy-from": "glock_19",
+    "type": "GUN",
+    "name": "defensive pistol",
+    "ammo": "ammo_pistol",
+    "description": "A modern pistol fit for duty, military service, or personal defense, with a detachable box magazine and a reliable action.  Though its chambering is capable of meeting FBI penetration minimums, the lack of a shoulder stock limits its utility.",
+    "magazines": [ [ "ammo_pistol", [ "pistol_mag",  "pistol_smg_mag" ] ] ]
+  },
+  {
+    "id": "pistol_pcc",
+    "copy-from": "cx4",
+    "type": "GUN",
+    "name": "survivalist carbine",
+    "ammo": "ammo_pistol",
+    "description": "These small carbines share ammunition and magazines with common pistols, offering a more controllable carbine than a normal rifle, while also reducing ammunition costs.  Because of their compatibility, they pair well with duty pistols, allowing one to transition to a more stable weapon without carrying extra ammo or magazines.",
+    "magazines": [ [ "ammo_pistol", [ "pistol_mag",  "pistol_smg_mag" ] ] ]
+  },
+  {
+    "id": "pistol_revolver",
+    "copy-from": "ruger_lcr_38",
+    "type": "GUN",
+    "name": "police revolver",
+    "ammo": "ammo_pistol",
+    "description": "This may not be an actual police firearm, but similar revolvers chambered for standard defensive calibers like this were a favorite of police departments for nearly a century, up until the 1986 Miami shootout.  Afterwards, the slow reloading and shooting of revolvers were considered liabilities; still, this model's accuracy and moderate recoil make for a serviceable sidearm, and there are no magazines for you to lose or damage.",
```

```suggestion
    "description": "Revolvers like this, chambered for standard defensive calibers, were a favorite of police departments for nearly a century, up until the 1986 Miami shootout.  Afterwards, the slow reloading and shooting of revolvers were considered liabilities; still, this model's accuracy and moderate recoil make for a serviceable sidearm, and there are no magazines for you to lose or damage.",
```
YMMV but I think that's more concise